app/code/Magento/CustomerImportExport/Model/Export/Address.php

```diff
@@ -27,6 +27,8 @@ class Address extends \Magento\ImportExport\Model\Export\Entity\AbstractEav
 
     const COLUMN_ADDRESS_ID = '_entity_id';
 
+    const COLUMN_CUSTOMER_ID = '_customer_id';
```

Comment:
Adding a constant is a backwards incompatible change (see http://devdocs.magento.com/guides/v2.1/ext-best-practices/extension-coding/backwards-compatible-development/index.html#documentation-of-backwards-incompatible-changes).

Suggestion by the core team:
* in this PR for `2.2-develop`, don't use a constant but hard-code the string
* in addition, create a PR for `2.3-develop` that uses the constant.

This way, we have a quicker fix for shops on 2.2 but we use your approach (with is the best practice!) for 2.3